mirrord/cli/src/config.rs

```diff
@@ -392,14 +392,27 @@ pub(super) struct PortForwardArgs {
     pub context: Option<String>,
 
     /// Mappings for port forwarding
+    /// Expected format is: '-L \[local_port:\]remote_ip_or_hostname:remote_port'
+    /// If the remote is given as an ip, this is parsed as soon as mirrord starts
+    /// Otherwise, the remote is assumed to be a hostname and lookup is performed in the cluster
+    /// after a connection is made to the target
```

Comment:
```suggestion
    /// after a local connection is accepted.
```
Also, can you separate sentences with periods? I don't think clap respects newlines when producing `--help` message from docs